source/Octopus.Tentacle.Kubernetes.Tests.Integration/KubernetesScriptServiceV1IntegrationTest.cs

```diff
@@ -241,20 +272,27 @@ Task ScriptCompleted(CancellationToken ct)
     }
 
     [Test]
-    public async Task WhenALongRunningScriptIsCancelled_TheScriptShouldStop()
+    [TestCase(ScriptType.Normal)]
+    [TestCase(ScriptType.Raw)]
```

Comment:
I've made all these tests run with both a raw script and a normal script but this makes the tests run quite a bit longer in teamcity. Is this ok? @APErebus

Reply:
Yep, that's fine.

These are no way the slowest tests we have, so I'm fine with that